expression/explain.go

```diff
@@ -170,6 +242,46 @@ func SortedExplainNormalizedExpressionList(exprs []Expression) []byte {
 	return sortedExplainExpressionList(exprs, true)
 }
 
+// SortedExplainExpressionList generates explain information for a list of expressions in order.
+// In some scenarios, the expr's order may not be stable when executing multiple times.
+// So we add a sort to make its explain result stable.
+func SortedExplainExpressionList4InList(exprs []Expression) []byte {
+	return sortedExplainExpressionList4InList(exprs, false)
+}
```

Comment:
This function is not used. Looks like it's useless?
And the same for the `normalized` parameter of `sortedExplainExpressionList4InList` and the `ExplainInfo4InList` method.

Reply:
Done